workflow/util/util.go

```diff
@@ -331,13 +332,36 @@ func ResumeWorkflow(wfIf v1alpha1.WorkflowInterface, workflowName string) error
 			wf.Spec.Suspend = nil
```

Comment:
Currently a Workflow as a whole will be unsuspended even if only a partial number of the suspended nodes are unsuspended with the resume command. Perhaps move this after the `for` loop to make sure to only unsuspend the workflow as a whole if _all_ nodes are unsuspended?